web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.utils.ts

```diff
@@ -114,21 +116,25 @@ const normalizeImpact = ({
 export const normalizePrimaryImpact = (input: NormalizeImpactInput) =>
   (input?.impact || input?.sanityImpact) && {
     ...normalizeImpact(input),
-    label: PRIMARY_IMPACT,
+    label: input._(PRIMARY_IMPACT),
```

Comment:
why using input here?

Reply:
To reuse existing parameters, the translator should probably be given as a separate parameter; I'll update that.